app/dev-portal/apis.md

```diff
@@ -37,13 +35,142 @@ faqs:
       [API Products](/api-products/) were used to create and publish APIs to classic (v2) Dev Portals. When the new (v3) Dev Portal was released, the API Products menu item was removed from the sidebar navigation of any {{site.konnect_short_name}} organization that didn't have an existing API product. If you want to create and publish APIs, you can create a new (v3) Dev Portal. To get started, see [Automate your API catalog with Dev Portal](/how-to/automate-api-catalog/).
 ---
 
-An API is the interface that you publish to your end customer. Developers register [applications](/dev-portal/self-service/) for use with specific APIs.
-
-As an API Producer, you [publish an OpenAPI or AsyncAPI specification](/dev-portal/publishing) and additional documentation to help users get started with your API.
-
-## Validation
+{:.success}
+> **How-to guides** <br>
+> This is a reference guide, you can also follow along with our tutorials: 
+>* [Automate your API catalog with Dev Portal](/how-to/automate-api-catalog/)
+>* [Automate your API catalog with Terraform](/how-to/automate-api-catalog-with-terraform/)
```

Comment:
```suggestion
{:.success}
> This is a reference guide, you can also follow along with our tutorials: 
>* [Automate your API catalog with Dev Portal](/how-to/automate-api-catalog/)
>* [Automate your API catalog with Terraform](/how-to/automate-api-catalog-with-terraform/)
```

Reply:
This makes it look like this, which feels kind of nice, what do you think?
<img width="915" height="302" alt="image" src="https://github.com/user-attachments/assets/9e1c57bd-2093-4f0a-a570-667bc3271ee6" />